backup-recovery.adoc

```diff
@@ -1,26 +1,101 @@
 
-* Private keys to on chain bitcoin and BSQ
-* Tor hostname, onion keys, state
-* Pending and closed trades, other trade statistics
+== Data Directory
 
-== How to Create a Backup
+=== Back Up
 
-You can create a complete backup of the entire wallet, in the *Account* menu and the *Backup* tab. *Select Backup Location* on your local computer, USB stick or CD, and click *Backup Now*. Careful though, the backup file is not encrypted, and thus anyone who has this file can take all your money! 
+All Bisq data is stored in one place:
 
-== How to Import a Backup
-[%hardbreaks]
-Shutdown the Bisq client and copy the entire backup data from your USB stick to 
-Mac OSX: `~/Library/Application Support/Bisq`
-Linux: `~/.local/share/Bisq`
-Windows 7,8,10: `C:\Users\username\AppData\Roaming\Bisq`
-Windows XP: `C:\Documents and Settings\username\Application Data\Bisq`
-Make sure that you merge and replace all the existing files. Importing the backup means that you delete the current state of the wallet, so make sure you have a second backup of the status quo. Upon restart of the Bisq client, the the backup account will automatically load.
+* macOS: `/Users/<username>/Library/Application Support/Bisq`
+* Linux: `/home/<username>/.local/share/Bisq`
+* Windows 7+: `C:\Documents and Settings\<username>\AppData\Roaming\Bisq`
+* Windows XP: `C:\Documents and Settings\<username>\Application Data\Bisq`
+
+**The safest, quickest, easiest, and most comprehensive thing you can do is to back up this whole folder at regular intervals.** You can either close Bisq and copy this folder to your backup location, or you can export a backup directly to your backup location while Bisq is running from the `Backup` tab of the `Account` screen.
+
+NOTE: Backup files are not encrypted. If you've set a wallet password, your wallet files will be encrypted, but all your other data will not be. We recommend encrypting the whole backup folder with something like gpg, 7-Zip, Cryptomator, etc.
+
+=== Restore
+
+First, make sure the Bisq program is closed. And just in case something goes wrong with the restore, back up the current state of your data directory (i.e., the `Bisq` folder).
+
+Then delete the whole `Bisq` folder and replace it with the `Bisq` folder from your backup.
+
+=== Details
+
+In your Bisq data directory, you will see a few items. The most notable file is `bisq.log`, which is the Bisq program log file. Checking it can be helpful when you have issues with the Bisq program.
+
+The other notable item is `btc_mainnet`, which is the main folder containing data for your payment accounts, Tor setup, and wallet.
+
+== Payment Accounts
+
+There are 2 components to payment accounts on Bisq: account details and account aging.
```

Comment:
Account age witness is only relevant for fiat accounts. For altcoin accounts it does not exits as there is no chargeback risk.

Reply:
Maybe add a link to the account age witness doc/spec for further details?